### PR TITLE
[homework #7] - js basics 2 by German Macchia

### PR DESCRIPTION
**Pull request by GermanMacchia**

### CODE REVIEW

File: homeworks/german-macchia/07-js-basics-2/basics-2.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
+// katas 1 - https://www.codewars.com/kata/5715eaedb436cf5606000381
+function positiveSum(arr) {
+  arr.push(0);
```
Comment (VitalikMelnyk) [inline homeworks/german-macchia/07-js-basics-2/basics-2.js:3]: You don't need to use the `push` method in this case.
Just add the initial value in `reduce` method as the second param

**Comment by VitalikMelnyk** [comment]
@GermanMacchia, I have checked your homework. One note about the first kata. Please verify)

File: homeworks/german-macchia/07-js-basics-2/basics-2.js (hunk 1 above)
Comment (GermanMacchia) [reply homeworks/german-macchia/07-js-basics-2/basics-2.js]: great! thanks!

**Comment by GermanMacchia** [comment]
Hey @VitalikMelnyk !
I fix the Katas 1.
I made some of the optionals too!

**Comment by VitalikMelnyk** [comment]
Well Done @GermanMacchia

**Review by VitalikMelnyk — APPROVED** [review]

**Pull request merged**